src_c/event.c

```diff
@@ -573,6 +603,12 @@ dict_from_event(SDL_Event *event)
             _pg_insobj(dict, "pinched", PyFloat_FromDouble(event->mgesture.dDist));
             _pg_insobj(dict, "num_fingers", PyInt_FromLong(event->mgesture.numFingers));
             break;
+        case SDL_MOUSEWHEEL:
+            /* https://wiki.libsdl.org/SDL_MouseWheelEvent */
+            obj = Py_BuildValue("(ii)", event->wheel.x, event->wheel.y);
+            _pg_insobj(dict, "pos", obj);
+            _pg_insobj(dict, "direction", PyInt_FromLong(event->wheel.direction));
```

Comment:
Should constants for this be added, ie MOUSEWHEEL_NORMAL and MOUSEWHEEL_FLIPPED?

Cf. https://wiki.libsdl.org/SDL_MouseWheelEvent

Reply:
Should the `which` field be added?

> which may be SDL_TOUCH_MOUSEID, for events that were generated by a touch input device, and not a real mouse. You might want to ignore such events, if your application already handles SDL_TouchFingerEvent.

If not, I think those events must be filtered, or else they will be received in pygame anyhow.